services/scabbard/libscabbard/src/service/shared.rs

```diff
@@ -196,3 +230,70 @@ impl ScabbardShared {
         Ok(true)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    use cylinder::{secp256k1::Secp256k1Context, VerifierFactory};
+    use splinter::service::{ServiceMessageContext, ServiceSendError};
+
+    /// Verifies that the `is_coordinator` and `coordinator_service_id` methods work properly.
+    #[test]
+    fn coordinator() {
```

Comment:
The test isn't clear as to why the asserts are correct.

Reply:
Added more details to the test's doc comment